enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/EnterpriseCoreEditionModule.java

```diff
@@ -193,6 +200,15 @@ public void registerEditionSpecificProcedures( Procedures procedures ) throws Ke
         life.add( coreServerModule.membershipWaiterLifecycle );
     }
 
+    static Predicate<String> fileWatcherFileNameFilter()
+    {
+        return Predicates.all(
+                fileName -> !fileName.startsWith( PhysicalLogFile.DEFAULT_NAME ),
+                filename -> !filename.startsWith( StoreUtil.BRANCH_SUBDIRECTORY ),
```

Comment:
@martinfurmanski  EnterpriseEditionModule can't have `branched` or `temp-copy` directories since it will not gonna copy it from anywhere, it's only pure single enterprise database and because of that is will inherit all the filters from community edition.

On the other part of comment: it's impossible for this edition to have branched directory right? (if so -  will remove that filter)